app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,58 @@
 <html>
   <head>
     <title>EMS</title>
+
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <a class="navbar-brand" href="/">EMS</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active" style="margin: 8px; margin-top: 0px">
+            <a class="nav-link">
+             <%= link_to "Home", root_path %>
+            </a>
+          </li>
+          <% if current_user %>
+            <li class="nav-item" style="margin: 8px; margin-top: 0px">
+              <a class="nav-link">
+               <%= link_to "My Profile", user_path(current_user) %>
+              </a>
+            </li>
+            <li class="nav-item" style="margin: 8px; margin-top: 0px">
+              <a class="nav-link">
+               <%= link_to "Sign Out", destroy_user_session_path, :method => :delete %>
+              </a>
+            </li>
+          <% else %>
+            <li class="nav-item" style="margin: 8px; margin-top: 0px">
+              <a class="nav-link">
+               <%= link_to "Sign In or Sign Up", new_user_session_path %>
```

Comment:
Shouldn't it be just "Sign In"?

Reply:
My project requires for the Login and Sign Up forms to be on the same page. What I can do is that I can rename it to Sign In and from there the user can be guided to Sign Up if required.